src/fastlane/release_actions/lib/fastlane/plugin/release_actions/helper/git.rb

```diff
@@ -0,0 +1,31 @@
+require 'fastlane_core/ui/ui'
+
+class Git
+  SEPARATOR = "=====END====="
+
+  def self.last_version
+    last_tag = run('git describe --tag', 'Could not find tag from HEAD')
+
+    2.times { last_tag, = last_tag.rpartition('-') }
+    last_tag
+  end
+
+  def self.log(from, to = 'HEAD')
```

Comment:
Are we sure that `from` and `to` will not be tainted with user input, making the command below unsafe to run?

Reply:
You're worried about a nested execution? I don't think that's possible here, but will test. Other concerns?

Reply:
So, passing this as a string will execute it in "shell form" which is actually a risk as input such as backticks and pipes does what they do in your shell. Using an array uses command form, which does not do this. I've updated this to command form.

Good catch!